src/thriveopedia/pages/wiki/mechanics/MechanicLinkButton.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Godot;
+
+/// <summary>
+///   Simple way to link mechanic pages from the mechanics root page
+/// </summary>
+public partial class MechanicLinkButton : Button
```

Comment:
Is there a reason this is a separate class to `PageLinkButton`?

Reply:
It'd make things clear if these two classes were renamed based on what they do rather than what they are used for.